src/actions/player/playerTrendsActions.js

```diff
@@ -63,9 +68,22 @@ export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch)
           start_time: match.start_time,
           win,
         });
-
         return dataList;
       }, []);
+
+      // Apply mean-smoothing by taking differences of the previously computed integral
+      for (let i = 0; i < trends.length; i += 1) {
+        if (i < trends.length - chunkSize) {
+          trends[i].value = (trends[i + chunkSize].value - trends[i].value) / chunkSize;
+        } else {
```

Comment:
Don't think we need this else block, ultimately we only want to do this this 480 times? (500 - chunkSize)  Moving mean isn't valid for the first 20 elements.